.travis.yml

```diff
@@ -1,18 +1,19 @@
+bundler_args: --without development --retry=3 --jobs=3
+cache: bundler
 language: ruby
 rvm:
+  - rbx-2
+  - jruby
+  - jruby-head
+  - ruby-head
```

Comment:
Since these builds take the longest to complete, they should be started first. The way it was working before, all the other builds would finish before the Rubinius build started and then it would take a few minutes for the entire build to complete. I wish I could draw easily insert a Gantt chart to illustrate this but I think you get the idea.